src/App.css

```diff
@@ -1,3 +1,8 @@
+body {
+  display: flex;
```

Comment:
for me, at the parent level components such as App, landingPage and ImageCapture page or any others it would be best to use CSS Grid with Grid layout to declare how the page lays out in mobile and in web. CSS Grid is very easy and powerful at the parent level but it can be a little overkill at times when you just want things to align like here. For now though, this should work, just food for thought.

Reply:
I had not thought about using CSS Grid but I see what you mean and I can switch it. Do you suggest having different grid selections for web and mobile using @media queries?

Reply:
yes, likely using media queries, can either add here or in a separate PR

Reply:
I've made the changes in this PR so that the leaderboard and category description are centered with CSS Grid. Right now theres duplicated code for the media queries since each component has its own css file but it can reduced if all of the css files are later combined.